config/application.yml.default

```diff
@@ -107,6 +107,7 @@ encrypted_document_storage_s3_bucket: ''
 event_disavowal_expiration_hours: 240
 feature_idv_force_gpo_verification_enabled: false
 feature_idv_hybrid_flow_enabled: true
+feature_new_device_alert_aggregation_enabled: true
```

Comment:
We should disable this in `production` (deployed) environments, in the `production:` block below.

Otherwise it will be enabled in production by default.

Reply:
Added that in.